### PR TITLE
[AAE-7376] Added process instance link in task details

### PR DESCRIPTION
**Pull request by tomgny**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
image

**What is the new behaviour?**
image

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
After click on process instance id, we are redirected to the process details.

### CODE REVIEW

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts [hunk 1]
```diff
@@ -230,6 +235,18 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
...
             ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PROCESS_INSTANCE_ID',
+                    value: this.processInstanceId,
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.PROCESS_INSTANCE_ID_DEFAULT'),
+                    key: 'processInstanceId',
+                    clickable: true,
+                    clickCallBack: () => {
+                        this.redirectToProcessDetails();
```
Comment (sivakumar414ram) [inline lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts:246]: Adf component should not handle the navigation at all the application that uses this header component is responsible to navigate on click of the process instance Id.

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts [hunk 2]
```diff
@@ -336,6 +353,14 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
         return filteredProperties ? filteredProperties.indexOf(cardItem.key) >= 0 : true;
     }
 
+    private redirectToProcessDetails() {
```
Comment (sivakumar414ram) [inline lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts:356]: This logic should be moved to the application level.

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts (hunk 1 above)
Comment (tomgny) [reply lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts]: logic moved to the application level

**Review by sivakumar414ram — COMMENTED** [review]
Just added minor comment, please check

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts [hunk 3]
```diff
@@ -230,6 +232,15 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
                     key: 'id'
                 }
             ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PROCESS_INSTANCE_ID',
```
[the remark this answers is not in the document]
Comment (tomgny) [reply lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts:237]: unit test added

**Review by sivakumar414ram — APPROVED** [review]

**Pull request merged**